x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.scss

```diff
@@ -0,0 +1,9 @@
+.relevanceTuningForm {
+  .relevanceTuningForm__item {
+    width: 100%;
+  }
+
+  .relevanceTuningForm__panel + .relevanceTuningForm__panel {
+    margin-top: $euiSizeM;
+  }
```

Comment:
SUPER optional suggestion - text decoration UI polish:

```suggestion
  }

  &__panel .euiAccordion__button {
    &:hover,
    &:focus {
      text-decoration: none;

      h3 {
        text-decoration: underline;
      }
    }
  }
```

Before:
<img width="553" alt="" src="https://user-images.githubusercontent.com/549407/108105464-81ee5f00-7041-11eb-83c3-81e0a50f10c8.png">

After:
<img width="569" alt="" src="https://user-images.githubusercontent.com/549407/108105472-83b82280-7041-11eb-997d-9c462f2730f2.png">

Note:
`h3` isn't super precise so feel free to apply a specific classname like `.relevanceTuningForm__fieldTitle`, up to you

Reply:
https://github.com/elastic/kibana/pull/90621/commits/805f5afc952c6ab0dc297308aa9b813c8629aeaf